lib/workers/repository/update/pr/changelog/github/index.ts

```diff
@@ -28,18 +28,21 @@ export async function getTags(
 ): Promise<string[]> {
   logger.trace('github.getTags()');
   try {
-    const url = `${endpoint}repos/${repository}/tags?per_page=100`;
-    const res = await http.getJson<GithubRestTag[]>(url, {
-      paginate: true,
-    });
-    const tags = res.body;
+    const tags = await queryTags(
+      {
+        registryUrl: endpoint,
+        packageName: repository,
+      },
+      http
```

Comment:
The part I'm not sure about: can we reuse results obtained via `http` object of different `hostType`?

Reply:
i don't think so, they can have different auth token. 🤔

Reply:
This means we need to add `hostType` as part of cache keys first

Reply:
Or maybe not the `hostType` by itself, but some sort of "auth fingerprint", so that different host types can share cache data when possible

Reply:
@rarkins Seems like we need your comments here